### ISSUE TITLE
Is it possible to cross between dataset formats and deep learning frameworks?

### ISSUE THREAD

**Issue by yuvval** [NVIDIA/DALI#74]
(1) Is it possible to cross between dataset formats and deep learning frameworks?
For example, 
(*) TFRecords with CAFFE2
(*) LMDB with Tensorflow

(2) If answer to Q1 is yes, could you provide an example of loading a dataset with the same format to Tensorflow and  CAFFE2 or pytorch? Currently, the Tensorflow example and pytorch example are very different, and it is hard to tell what parts of the code are general, and what are customized for the deep-learning framework.

**Comment by JanuszL** [comment]
Hi,
Please check docs/examples/tensorflow/tensorflow-resnet50-various-readers.ipynb.
It is TF based example that shows how to use different readers. You can see what results they return and how to obtain images and labels.
If you put readers form mentioned example to PyTorch based one, it should work without any additional effort (replace HybridPipe with any reader like MXNetReaderPipeline, FileReadPipeline or TFRecordPipeline).

**Comment by yuvval** [comment]
Thank you. Sorry, not sure how I missed that file.

Also I found a great and concise example on [this NVIDIA presentation](http://on-demand.gputechconf.com/gtc/2018/presentation/s8906-fast-data-pipelines-for-deep-learning-training.pdf). Here the same reader is piped either to TF, CAFFE2 or MXNET.
